### PR TITLE
align this readme with the core one.  formatting and layout is now th…

### PR DESCRIPTION
**Pull request by jacobsalmela**
…e same.  also added a ton of updated screenshots.  finally, updated donation and affiliate links

Signed-off-by: Jacob Salmela <jacob.salmela@pi-hole.net>

**By submitting this pull request, I confirm the following:** `{please fill any appropriate checkboxes, e.g: [X]}`

`{Please ensure that your pull request is for the 'devel' branch!}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#git-commit---signoff))

---

**What does this PR aim to accomplish?:**

- Update donation and affiliate links
- Add links to trademark guidelines
- Added several new screenshots
- Added updated API example
- Add a new project

**How does this PR accomplish the above?:**

Made the appropriate edits.

**What documentation changes (if any) are needed to support this PR?:**
The README _is_ documenation

---
* You must follow the template instructions. Failure to do so will result in your pull request being closed.
* Please respect that Pi-hole is developed by volunteers, who can only reply in their spare time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,186 @@
...
+## Black and white lists
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/blacklist.png" alt="Blacklist"></a>
+</br></br>
+<img src="https://pi-hole.github.io/graphics/Screenshots/whitelist.png" alt="Whitelist"></a>
+</p>
+
+## The ability to easily manage and configure Pi-hole features
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/dashboard-pihole-repo.png" alt="Settings"></a>
```
Comment (Mcat12) [inline README.md]: This image isn't showing up for me

Comment (jacobsalmela) [reply README.md]: Oops.  I never finished that one.  It's a gif of the different settings page.  Good catch.  I'll fix that

File: README.md [hunk 2]
```diff
@@ -1,32 +1,186 @@
...
+## Black and white lists
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/blacklist.png" alt="Blacklist"></a>
+</br></br>
+<img src="https://pi-hole.github.io/graphics/Screenshots/whitelist.png" alt="Whitelist"></a>
+</p>
+
+## The ability to easily manage and configure Pi-hole features
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/dashboard-settings.gif" alt="Settings"></a>
```
Comment (Mcat12) [inline README.md]: The first 3 frames of the gif have the DNSSEC checkbox at the bottom of the page.

Comment (jacobsalmela) [reply README.md]: OK this should be fixed now.

**Pull request merged**